src/order/galois_connection.lean

```diff
@@ -200,20 +200,24 @@ end
 end partial_order
 
 section order_top
-variables [partial_order α] [preorder β] [order_top α] [order_top β] {l : α → β} {u : β → α}
-  (gc : galois_connection l u)
-include gc
```

Comment:
Why not keep `l`, `u` and `gc` into the variables? Is it to keep this specific order of parameters

Reply:
Yes, I prefer to have TC arguments before other arguments.